dropbox/dropbox_uploader.sh

```diff
@@ -346,7 +346,7 @@ function check_http_response
     esac
 
     #Checking response file for generic errors
-    if grep -q "HTTP/1.1 400" "$RESPONSE_FILE"; then
+    if grep -q "^HTTP/[12].* 400" "$RESPONSE_FILE"; then
```

Comment:
all the individual http response handling makes me wonder if this wouldn't be best extracted somehow.

Reply:
Probably, but I don't want to diverge too far from andreafabrizi/Dropbox-Uploader .

Reply:
fair enough.  have we rebased on top of that since forking this?  are there other updates there we should have?

Reply:
The HTTP/2 update is verbatim from that repo; it's the only change there since the import.

The printf format fix is mine; I've submitted it as a PR there as well.

Reply:
We were discussing at some point directly using dropbox java api in jibri ... But never got it prioritized